crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -955,9 +994,7 @@ pub fn queue_mesh_view_bind_groups(
     render_device: Res<RenderDevice>,
     mesh_pipeline: Res<MeshPipeline>,
     shadow_samplers: Res<ShadowSamplers>,
-    light_meta: Res<LightMeta>,
-    global_light_meta: Res<GlobalLightMeta>,
-    fog_meta: Res<FogMeta>,
+    (light_meta, global_light_meta, fog_meta): (Res<LightMeta>, Res<GlobalLightMeta>, Res<FogMeta>),
```

Comment:
Tentatively using the [tuple-based solution](https://github.com/bevyengine/bevy/pull/8704#discussion_r1208894919) proposed by @nicopap instead of `SystemParam`, if that's okay. cc @alice-i-cecile